packaging/debian/rules

```diff
@@ -22,19 +22,28 @@ override_dh_auto_install:
 	install -p -m 0644 instance_configs.cfg debian/google-guest-agent/usr/share/google-guest-agent
 	install -d debian/google-guest-agent/lib/systemd/system
 	install -p -m 0644 gce-workload-cert-refresh.timer debian/google-guest-agent/lib/systemd/system/
+	if [[ -d google-guest-agent ]]; do\
```

Comment:
is this the right path? I believe guest-agent and google-guest-agent repos would be side-by-side and the package build happens from root of guest-agent repo

Reply:
With how the startupscripts are set up for the build, this is unfortunately not the case. So the workaround right now is to check in the `google-guest-agent` repo within the `guest-agent` repo and go from there.

Reply:
startup scripts are not yet configured to pull extra repo, can't we make it clone the way we want? It would be weird to have one repo within another

Reply:
Resolving